### DISCUSSION TITLE
Ownership and `~/.gitconfig`

### DISCUSSION THREAD

**Discussion by Xophmeister** [github-linguist/linguist#7076] [Q&A] [answered]
For what technical reason does GitHub Linguist (that is: the CLI) insist on having to run as the same owner that owns (in terms of the filesystem) the repository that it's checking? I notice that it's trying to access `~/.gitconfig`: is this related?

**Comment by lildude** [comment, marked as answer]
There isn't anything specifically in Linguist that requires this however it is designed to run against a repo as repos are stored on disk as bare repos at GitHub so we can access the raw files directly. I believe the requirement you are setting is actually a git enforcement.

**Reply by Xophmeister** [reply]
Interesting :+1: That would explain why I couldn't find the error message when `grep`ing through the code. Thanks :pray:
